### PR TITLE
Changes for e2e to work

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
Made small changes to swift for e2e to work

cc @danmcp 
FYI @awalkaradi95

### CODE REVIEW

File: openshift/pman-swift-publisher/put_data.py [hunk 1]
```diff
@@ -91,6 +87,5 @@ def storeData(self, **kwargs):
 
 
 if __name__ == "__main__":
-
-    obj = SwiftStore()
-    obj.storeData(path= os.environ.get('SWIFT_KEY'))
+    swiftStore = SwiftStore()
+    swiftStore.storeData(path= os.environ.get('SWIFT_KEY'))
```
Comment (danmcp) [inline openshift/pman-swift-publisher/put_data.py]: remove the space after =

**Comment by rudolphpienaar** [comment]
Let me know when OK to merge...!

File: openshift/pman-swift-publisher/put_data.py [hunk 2]
```diff
@@ -49,34 +43,36 @@ def zipdir(self, path, ziph, **kwargs):
...
+                #TODO: @ravig - Change this so that this is scalable.
                 zippedFileContent = f.read()
         finally:
-            os.remove('/share/outgoing/ziparchive.zip')
+            ziphandler.close()
+            os.remove('/tmp/ziparchive.zip')
 
-        swiftHandler = SwiftHandler()
-        self.swiftConnection = swiftHandler._initiateSwiftConnection()            
+        swiftHandler=SwiftHandler()
```
[the remark this answers is not in the document]
Comment (ravisantoshgudimetla) [reply openshift/pman-swift-publisher/put_data.py]: Weird, my pep8 didn't complain, thanks for catching it.

File: openshift/pman-swift-publisher/put_data.py (hunk 1 above)
Comment (ravisantoshgudimetla) [reply openshift/pman-swift-publisher/put_data.py]: Ack. This time PEP8 complained and I somehow ignored it.

**Comment by ravisantoshgudimetla** [comment]
@danmcp - Addressed your comments PTAL.

**Pull request merged**